### ISSUE TITLE
stack creation failed- "...IAM policy that is too large"

### ISSUE THREAD

**Issue by ManishMithaiwala** [Senzing/aws-cloudformation-ecs-senzing-stack-basic#10]
The cloudformation script  [(aws-cloudformation-database-cluster)](https://github.com/Senzing/aws-cloudformation-database-cluster/tree/1.0.0) succeeded without error, however the [senzing-stack-basic](https://github.com/Senzing/aws-cloudformation-ecs-senzing-stack-basic/tree/1.0.3) failed with several errors. The **first encountered** error is seen in the figure. The scripts were run with Administrator level access. Based on the error, we investigated the senzing-db-poc-IamRoleLambda created in aws-cloudformation-database-cluster, and found there were no policies attached. As an experiment we manually added an IAMFullAccess policy and retried. _This time the senzing-stack-basic script ran successfully._ 

It is uncertain exactly what policies are needed for senzing-db-poc-IamRoleLambda, but they need some IAM roles, or 
perhaps it should just not be empty.

**To Reproduce**
Deploy the two cloudformation scripts in AWS. Upon failure of the senzing-stack-basic look for error as in the figure. 

![image (6)](https://user-images.githubusercontent.com/10515685/119044266-fc53b580-b987-11eb-9fb4-b80e28951bc1.png)

**Comment by docktermj** [comment]
Hi @ManishMithaiwala!

An interesting discovery.  I've run the scenario many times and haven't encountered this issue.   What AWS region are you running in?

**Comment by dennisjunior111** [comment]
We are running on US East (N. Virginia) us-east-1.

**Comment by docktermj** [comment]
...geez, that's where I do most of my testing.   @antaenc We may have to compare our master account to theirs to figure out what is different so we can figure out if code is needed or if this is a documentation issue.

**Comment by dennisjunior111** [comment]
Today we started from scratch and deleted both the database and basic stacks. After recreating them again, they both ran without issue. I am pretty sure we set them up in the same way and same permissions, so I would presume it was an issue with AWS provisioning dependent resources or something. Tricky as from the errors it is hard to tell if it was an issue with the stack's configuration or AWS itself; especially when we aren't the ones to write the stack ourselves. Either way, the problem is fixed and this issue can be closed.
